### DISCUSSION TITLE
How to remove the shaded area of a image

### DISCUSSION THREAD

**Discussion by tonyfkchu** [dlemstra/Magick.NET#757] [Help]
Dear Expert,

How can I remove the shaded area of the following image with IM.net?

<img width="304" alt="shaded" src="https://user-images.githubusercontent.com/6896544/95753056-3579c800-0ccb-11eb-86c2-bd272fa9d2f2.PNG">

**Comment by dlemstra** [comment]
Can you help us with this @fmw42? I can translate it to C#.

**Comment by fmw42** [comment]
Here are 3 possibilities:



convert img.png -colorspace gray -negate -lat 10x10+5% -negate x.png




convert img.png -colorspace gray -negate -lat 20x20+5% -negate x2.png





convert img.png \( +clone -blur 0x5 \) +swap -compose divide -composite -contrast-stretch 3x0% x3.png




Sorry, I had no luck with the dotted lines issue.

Fred

**Comment by tonyfkchu** [comment]
How can convert this to .with  .Net APi ?

convert img.png \( +clone -blur 0x5 \) +swap -compose divide -composite -contrast-stretch 3x0% x3.png

**Reply by dlemstra** [reply]
That command would translate to this:

```C#
using (var image = new MagickImage("img.png"))
{
    using (var clone = image.Clone())
    {
        clone.Blur(0, 5);
        clone.Composite(image, CompositeOperator.DivideDst);
        clone.ContrastStretch(new Percentage(3), new Percentage(0));
        clone.Write("x3.png");
    }
}
```
